python/cudf/cudf/_lib/copying.pyx

```diff
@@ -735,3 +737,120 @@ def segmented_gather(Column source_column, Column gather_map):
 
     result = Column.from_unique_ptr(move(c_result))
     return result
+
+
+cdef class PackedColumns:
```

Comment:
will want a docstring for this

Reply:
Is the new docstring suitable? Not sure if I should be putting examples since this probably isn't intended to be a public API right now